packages/content/src/enrichments.ts

```diff
@@ -596,7 +598,10 @@ export const getLayerContent = (
     return;
   }
   const { type, name, description } = layer;
-  const layerContent = { ...content, layer, type };
+  // get family and normalized type based on layer type
+  const family = getFamily(type);
+  const normalizedType = normalizeItemType({ ...content.item, type });
+  const layerContent = { ...content, layer, type, family, normalizedType };
```

Comment:
I'm pretty sure this is the only change in this whole diff that's not just shufflin' from -content to -common